ClassySchedule/src/main/java/samples/Main.java

```diff
@@ -62,19 +63,113 @@ public void handle(ActionEvent e)
 
     }
 
-    public void login() {
+
+
+    public void adminLogin() {
 
 
         User test = new Admin("Bob", "Test", 123123,6,6,"easilyCrackable");
         Text text = new Text();
-        text.setText(test.getFirstName() + " " + test.getLastName());
+        text.setText(test.getFirstName() + " " + test.getLastName()+"'s homepage (Admin)");
 
         GridPane screen = new GridPane();
         screen.addRow(0, text);
-        Scene mainApplication = new Scene(screen, 500, 300);
+
+        // For adding new Users
+        screen.add(new Text(),0,1);
+        // Elements for adding a new user
+        Button newUser = new Button("Add new user");
+        EventHandler<ActionEvent> addNewUser = new EventHandler<ActionEvent>() {
+            public void handle(ActionEvent e)
+            {
+                addNewUser();
+            }
+        };
+        newUser.setOnAction(addNewUser);
+        screen.add(newUser, 0,2);
+
+        // Adding a new course offering
+        Button newCourse = new Button("Add new course");
+        EventHandler<ActionEvent> addNewCourse = new EventHandler<ActionEvent>() {
+            public void handle(ActionEvent e)
+            {
+                addNewCourse();
+            }
+        };
+        newCourse.setOnAction(addNewCourse);
+        screen.add(newCourse, 0,3);
+
+        Button viewSchedule = new Button("View Schedule");
+        screen.add(viewSchedule, 0,4);
+
+
+        Scene mainApplication = new Scene(screen, 1000, 700);
         primaryStage.setScene(mainApplication);
     }
 
+    public void addNewCourse() {
+        GridPane screen = new GridPane();
+
+        EventHandler<ActionEvent> event = new EventHandler<ActionEvent>() {
```

Comment:
I think we should also figure out how to use controllers to build button and other items functionality. Doing it like this leads to messy code